### ISSUE TITLE
BCEWithLogitsLossFlat() as loss function in 06_multicat.ipynb

### ISSUE THREAD

**Issue by piegu** [fastai/fastbook#172]
Hello,

In the paragraph **Binary Cross-Entropy** in the notebook [06_multicat.ipynb](https://github.com/fastai/fastbook/blob/master/06_multicat.ipynb), it is written:

> We don't actually need to tell fastai to use this loss function (although we can if we want) since it will be automatically chosen for us. fastai knows that the DataLoaders has multiple category labels, so it will use nn.BCEWithLogitsLoss by default.

In fact, the loss function by default in this case is not `nn.BCEWithLogitsLoss` but [BCEWithLogitsLossFlat()](https://dev.fast.ai/layers#BCEWithLogitsLossFlat).

**Comment by jph00** [comment]
It's not really a significant different.
